bbtest/hosts.py

```diff
@@ -56,6 +60,12 @@ def run(self, cmd, *args):
     def put(self, local, remote):
         return shutil.copyfile(local, remote)
 
+    def os_remove(self, path):
```

Comment:
Do we need this and `rmtree`? It seems like one is enough, and the latter lets you remove a directory even if it's not empty.